src/lib/common/sol-mainloop.c

```diff
@@ -76,6 +76,10 @@ extern void sol_comms_shutdown(void);
 extern int sol_update_init(void);
 extern void sol_update_shutdown(void);
 #endif
+#ifdef USE_UPDATE
```

Comment:
Nope, really. It was wrong before ;)
Maybe I should do this on a separated commit.